lib/ansible/modules/windows/win_uri.py

```diff
@@ -148,6 +148,12 @@
       secure web request.
     type: str
     version_added: '2.5'
+  use_default_credentials:
+    description:
+    - Set the the ability of UseDefaultCredentials.
```

Comment:
typo 'the the'.  Suggest 'Use default windows credentials for authentication.  Note that you must be using kerberos, ntlm or credssp for this to take effect.

(see https://docs.microsoft.com/en-us/dotnet/api/system.net.credentialcache.defaultcredentials?view=netframework-4.8#remarks)

Reply:
changed